code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -152,8 +152,9 @@
 //CIG PACK//
 ////////////
 /obj/item/storage/fancy/cigarettes
-	name = "Trans-Stellar Duty Free cigarette packet"
+	name = "Trans-Stellar Duty Frees cigarette packet"
```

Comment:
```suggestion
	name = "Trans-Stellar Duty Free cigarette packet"
```
Duty Free is the accurate term

Reply:
it's the name of the cigarettes, not the packet